tests/KeyDBTest.php

```diff
@@ -21,19 +21,19 @@ public function testComputeKeyId($keyMeta, $want)
     public function computeKeyIdProvider()
     {
         return [
-          "case 1" => [
-            0 => [
-                  "keyid_hash_algorithms" => ['sha256', 'sha512'],
-                  "keytype" => "ed25519",
-                  "keyval" => ["public" => "edcd0a32a07dce33f7c7873aaffbff36d20ea30787574ead335eefd337e4dacd"],
-                  "scheme" => "ed25519"
+            "case 1" => [
+                0 => [
+                    "keyid_hash_algorithms" => ['sha256', 'sha512'],
+                    "keytype" => "ed25519",
+                    "keyval" => ["public" => "edcd0a32a07dce33f7c7873aaffbff36d20ea30787574ead335eefd337e4dacd"],
+                    "scheme" => "ed25519"
```

Comment:
Should we be testing for trailing comma too?

Reply:
Hmm, I thought I added a rule for the trailing comma. Will look into this later.

Reply:
So it did add a missing trailing comma in one place where the closing bracket was on a line by itself, but not to this line here. I checked and there is a long open PHPCS issue about this problem: https://github.com/squizlabs/PHP_CodeSniffer/issues/582

PEAR doesn't seem to have anything to say about it either.

So, for this to be detected all the time, we'd need to add some fork of the Drupal array sniff or install another of the various third-party sniffs created related to it.

For now my suggestion is to manually add the trailing comma and file an issue to improve detection for missing trailing commas on multiline arrays.

Reply:
I pushed b51bb91 to add the missing trailing comma manually for now.

Reply:
Created a followup https://github.com/php-tuf/php-tuf/issues/38 to address this problem, so I propose we resolve this there and fix what we can here, since this can be solved with official sniffs. Thanks!